### ISSUE TITLE
Code actions for adding dependencies

### ISSUE THREAD

**Issue by mlutze** [flix/flix#6273]
to Flix toml

**Comment by magnus-madsen** [comment]
Would this require global knowledge of all packages?

**Comment by mlutze** [comment]
No, I'm not thinking autocompleting package names, but rather giving the template for adding the dependency. I can never remember the syntax.

**Comment by magnus-madsen** [comment]
> No, I'm not thinking autocompleting package names, but rather giving the template for adding the dependency. I can never remember the syntax.

Maybe this is best left for some TOML-tooling?

**Comment by magnus-madsen** [comment]
We cannot practically do this. Flix LSP is not registered for TOML and doing so is not what we want.
